### PR TITLE
Add class gifts

### PR DESCRIPTION
**Pull request by darkodosenovic**
:art:

Moving one of the gift classes to here

### CODE REVIEW

**Comment by sineade** [comment]
:book:

File: app/assets/stylesheets/layout_components/content-toggle.scss [hunk 1]
```diff
@@ -34,3 +34,17 @@
     display: block;
   }
 }
+
+.edit_gift{
+  .associate-expandable{
+    display: none;
+  }
+
+  .join-checkout{
```
Comment (sineade) [inline app/assets/stylesheets/layout_components/content-toggle.scss]: would we be able to actually get rid of this class and in your other branch just change it to use .micetype where this class was being applied?

**Comment by sineade** [comment]
:hankey:

**Comment by darkodosenovic** [comment]
:wave:

**Comment by sineade** [comment]
:art: :art: :art:

**Pull request merged**